frontend/components/DashboardApp.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+function DashboardApp() {
```

Comment:
It's much common to use arrow functions in React for func. components
ex. 
`const DashboardApp = () => {}`